pkgs/applications/networking/instant-messengers/cinny-desktop/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, lib
+, dpkg
+, fetchurl
+, autoPatchelfHook
+, openssl
+, gnused
+, webkitgtk
+, makeWrapper
+, disableCompositing ? false # https://bugs.webkit.org/show_bug.cgi?id=180739
```

Comment:
IMO this shouldn't be an option. A user could utilize `postFixup` to regenerate wrapper if he chooses to do so because having an NVIDIA card with proprietary drivers is a niche case. 

However, I would really like to hear a second opinion on this.

Reply:
@SuperSandro2000 could you please share your thoughts on this?

Reply:
I don't think we are doing this anywhere else. I would recommend to set the environment variable in ``environment.variable`` to fix this globally.

Reply:
I have now removed it and set it in my global environment variables nixos setting.